includes/Context.php

```diff
@@ -164,11 +164,11 @@ public function get_reference_site_url() {
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param int|\WP_Post $post  Optional. Post ID or post object. Default is the global `$post`.
+	 * @param int|\WP_Post $post  Post ID or post object. Optional. Default is the global `$post`.
 	 *
-	 * @return string|false The  Reference permalink URL or false if post does not exist.
+	 * @return string|false The reference permalink URL or false if post does not exist.
 	 */
-	public function get_reference_permalink( $post = 0 ) {
+	public function get_reference_permalink( $post = false ) {
```

Comment:
setting as `false` by default, 0 feels like the wrong value here

Reply:
Why that? `0` is also "falsey", and `false` does not match the method signature. I think this should remain as it is.

Reply:
```suggestion
	public function get_reference_permalink( $post = 0 ) {
```